examples/module1/lesson1/solver/OperationButtonsGrid.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { add, divide, multiply, substract } from './functions';
+import OperationButton from './OperationButton';
+
+interface OperationButtonsGridProps {
+  firstNumber: number;
+  secondNumber: number;
+  setResult: React.Dispatch<React.SetStateAction<number | string>>;
+}
+
+const OperationButtonsGrid: React.FC<OperationButtonsGridProps> = ({ firstNumber, secondNumber, setResult }) => {
+  return (
+    <div className="grid grid-cols-4 gap-x-4 my-4">
+      <OperationButton operation={add} label="+" firstNumber={firstNumber} secondNumber={secondNumber} result={setResult} />
+      <OperationButton operation={substract} label="-" firstNumber={firstNumber} secondNumber={secondNumber} result={setResult} />
+      <OperationButton operation={multiply} label="*" firstNumber={firstNumber} secondNumber={secondNumber} result={setResult} />
+      <OperationButton operation={divide} label="/" firstNumber={firstNumber} secondNumber={secondNumber} result={setResult} />
+    </div>
+  );
+};
```

Comment:
Zastanawiam się, czy ten komponent nie jest nadmiarowy w tym konkretnym zadaniu. Realnie nie ma on żadnej konkretnej odpowiedzialności (zbiera buttony), no i pytanie czy można byłoby ten komponent użyć w jakimś innym kontekście? No bo jeśli nie, to zyskujesz kolejną warstwę przekazywania propsów i funkcji tylko po to, żeby na samym dole trzewa, w `OperationButton` wywołać metodę do obliczania a i b.